File: OrbitalWatch/wear/src/main/java/com/dubul/dire/orbitalwatch/State.java
package com.dubul.dire.orbitalwatch;

/**
 * Created by domin on 8 Aug 2016.
 */
public abstract class State {
    public final static float PI = (float) Math.PI;

    public abstract void init();

    public abstract void update(float delta);

    public abstract void render(Painter c);

    public abstract void ambientRender(Painter c);

}

File: lwjgl3-test/src/state/State.kt
package state

import renderer.Painter

/**
 * Created by domin on 22 Mar 2017.
 */

public abstract class State {
    //    public int width = G.WIDTH;
    //    public int height = G.HEIGHT;
    //    int centerX = width/2;
    //    int centerY = height/2;
    internal var PI = Math.PI.toFloat()
    public var initialised = false

    abstract fun init()

    abstract fun update(delta: Float)

    abstract fun render(g: Painter)


}
